exchange/bidder_test.go

```diff
@@ -974,6 +932,28 @@ func TestMakeExt(t *testing.T) {
 				Status:         999,
 			},
 		},
```

Comment:
Could you please keep the original `"Request & Response - No Error"` test in addition to the new `"Request & Response - No Error with Authorization removal"` test? I think it's useful to track down possible implementation errors.